src/MagnumPlugins/AssimpImporter/Test/CMakeLists.txt

```diff
@@ -87,7 +81,12 @@ corrade_add_test(AssimpImporterTest AssimpImporterTest.cpp
         y-up.dae
         z-up.dae)
 target_link_libraries(AssimpImporterTest PRIVATE Assimp::Assimp)
-target_include_directories(AssimpImporterTest PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>)
+target_include_directories(AssimpImporterTest PRIVATE
+    ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>
+    # The test needs access to configureInternal.h written by
+    # AssimpImporter. The dynamic library doesn't get linked to
+    # and hence doesn't get the binary dir in the include dirs.
+    ${PROJECT_BINARY_DIR}/src)
```

Comment:
Not a huge fan of this but I didn't see a better way to let the tests access headers in the binary dir of `AssimpImporter`

Reply:
This is fine, I'm doing this regularly :)